src/utils/getFunctions.jsx

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export function getArticles() {
+  return axios
+    .get("https://nc-news-app.onrender.com/api/articles")
```

Comment:
I think you should create a baseURL using axios.create so that you can edit it there if you ever need to.